src/managers/TileManager.ts

```diff
@@ -47,6 +48,25 @@ class TileManager {
         );
     }
 
+    public getStartingTiles(): Tile[] {
+        const session: Nullable<SessionDTO> = GeneralManager.getSession();
+        assert(session);
+        const tiles: Tile[] = [];
+        const random: seedrandom.PRNG = seedrandom(session.seed);
+        const veins: int[] = seededShuffle([9, 10, 10], session.seed); // 9 is gold, 10 are coal veins
+
+        for (const item of preplacedTiles) {
+            const tile = new Tile(random, item.type || veins.pop()!);
```

Comment:
@C0DECYCLE I know you are not a fan of the "!" instead of assert but I think in this situation it makes for cleaner code, as we know it will never be null. It allows for a sexy oneliner 😎

Reply:
oke i can live with it :)